packages/bpk-component-accordion/src/BpkAccordion.js

```diff
@@ -29,26 +29,36 @@ const getClassName = cssModules(STYLES);
 
 type Props = { children: Node, className: ?string };
 
+export const BpkAccordionContext = createContext({ onDark: false });
```

Comment:
What's the justification for creatingContext to pass `onDark` down instead of just passing the props without context similar to other components?

Reply:
I thought it would be a bit clunkly with the `children` prop. However, happy to modify.